docs/config-lnd-integrated.md

```diff
@@ -310,7 +310,7 @@ as it is the default directory (`~/.lnd` on Linux).
 
 ### Example `loop` command
 
-This is where things get a bit tricky. Because as mentioned above, `loopd` also
+This is where things get a bit tricky. As mentioned above, `loopd` also
```

Comment:
I think instead it should be: `This is where things get a bit tricky, because as mentioned above....`